build.sh

```diff
@@ -401,6 +410,15 @@ build_one() {
                 INCLUDE_PATH="${root}/include" \
                 LIBRARY_PATH="${root}/lib" install
         ;;
+    libzip)
+	do_cmake \
```

Comment:
That post talks about the opposite case: building with CMake on Linux.  According to https://github.com/nih-at/libzip/issues/1#issuecomment-209377105, Autoconf should work for Windows builds, but I'm seeing the same problem with `dlfcn.h`.